elements/pl-matrix-component-input/pl-matrix-component-input.py

```diff
@@ -210,9 +213,77 @@ def render(element_html, data):
     return html
 
 
+def parse_entry(name, data, allow_fractions):
```

Comment:
Maybe this should be embedded in `pl` module as it is now shared across multiple elements?

Reply:
Yeah, after looking at the other functions in there I think that would be best in terms of not repeating code